src/nidcpower/metadata/attributes.py

```diff
@@ -36,7 +33,6 @@
     },
     1050203: {
         'access': 'read only',
-        'channel_based': False,
         'documentation': {
             'description': 'Indicates the number of channels that NI-DCPower supports for the instrument that was chosen when  the current session was opened. For channel-based attributes, the IVI engine maintains a separate  cache value for each channel.'
```

Comment:
This doesn't support `instruments` rep cap?

More broadly, are the supported rep cap changes already reviewed in internal metadata?